### ISSUE TITLE
Search + Visual

### ISSUE THREAD

**Issue by MarioRicalde** [SublimeText/VintageEx#22]
Is it possible that with the Search functionality you added to VintageEx add some way to "highlight to match"?

On Vim you can enter visual mode and use search to highlight till the given match.

**Comment by guillermooo** [comment]
It looks possible, but I'm not sure how well it'd play with multiple sels. I might give it a try. (In any case, the idea is to move the search functionality to Vintage at some point.)

**Comment by MarioRicalde** [comment]
I don't think we need to worry about multiple selections, the idea is that visual + search to jump to the given search.

If you need any help with testing let me know, I'm dying to see this feature implemented, it's really helpful with refactoring.
